download-maven-plugin/src/main/java/com/googlecode/WGet.java

```diff
@@ -367,11 +367,15 @@ private void doGet(File outputFile) throws Exception {
     }
     // TODO: this should be retrieved from wagonManager
     com.googlecode.ConsoleDownloadMonitor downloadMonitor = new com.googlecode.ConsoleDownloadMonitor();
```

Comment:
Instantiating the download monitor could also be in the condition succeeded block